ts/components/LabelledItem.tsx

```diff
@@ -23,23 +23,23 @@ const styles = StyleSheet.create({
   },
   bottomLine: {
     borderBottomWidth: 1
-  },
-  inputStyle: {
-    height: 50,
-    paddingLeft: 5,
-    paddingRight: 5,
-    fontSize: 17,
-    color: "#5C6F82",
-    flex: 1
   }
 });
 
 type Props = Readonly<{
+  type: "masked";
```

Comment:
perhaps, for readability, give an intermediate name to these two types before defining their union?